src/app/components/elements/UserList.jsx

```diff
@@ -23,18 +23,19 @@ class UserList extends React.Component {
 
     render() {
         const { state: { historyIndex } } = this;
+        const account = this.props.account;
         const users = this.props.users;
         const title = this.props.title;
 
         let idx = 0;
         let user_list = users.map(user => (
-            <UserListRow user={user} key={idx++} />
+            <UserListRow account={account} user={user} key={idx++} />
```

Comment:
this is a bit out of scope for the bugfix going on here but can't we use user ids as a key?